### ISSUE TITLE
Allow use of local url (mainly for docker usage)

### ISSUE THREAD

**Issue by JigSawFr** [jkirkcaldy/plex-utills#97]
Hi there,
In config, we are unable to use url of type:
http://plex:32400 for Plex for eg. (same logic for Tautulli)
In a docker environment we can reach container in the same network without the need to go on internet :)
So flows remain local, if you can accept these values, it could be great,
Thanks !

**Comment by JigSawFr** [comment]
It can work atm by editing directly the DB with something like this:
```
UPDATE plex_utills SET plexurl = REPLACE(plexurl,'http://plex:url','http://plex:32400');
UPDATE plex_utills SET tautulli_server = REPLACE(tautulli_server,'https://tautulli.url','http://tautulli:8181');
```
But not future proof :)

**Comment by jkirkcaldy** [comment]
if you use a local IP address it stays local and doesn't go via the internet. I use a url and because I have a local DNS server that resolves it it doesn't go out over the internet either.

**Comment by JigSawFr** [comment]
Yeah but that's not what I aim, internal IP can change as container are auto created on new version available.
So only possibility is to us docker dns name of container

**Comment by Anon0511** [comment]
Basically, this just works fine. No need for the workaround in the database.
The address is <containerName>.<networkName> i.E http://tautulli.plex:8181/
Although it would be nice if the container Name alone could work.

**Comment by jkirkcaldy** [comment]
This is fixed in the latest :dev container

`http://container:port` should now work
